notifications/server.go

```diff
@@ -82,3 +82,59 @@ func (s *server) SubscribeNotifications(
 	log.Printf("%v was successfully registered for notifications on url %s", pubkey, request.Url)
 	return &SubscribeNotificationsReply{}, nil
 }
+
+func (s *server) UnsubscribeNotification(
```

Comment:
This function in not in the proto service definition. There's 2 options I think:
1) Add it to the service definition. If we do that, we need a PR in breez server as well.
2) reuse the same encrypted request on SubscribeNotifications and check which function is being called by decrypting the payload.